CHANGELOG.md

```diff
@@ -139,7 +139,7 @@ If all went well, you should now see PHPCompatibility listed again in the list o
 
     Install type | Old path | New path
     ------------ | -------- | ---------
-    Composer     | `vendor/wimg` | `vendor/wimg/php-compatibility`
+    Composer     | `vendor/wimg` | `vendor/phpcompatibility/php-compatibility`
```

Comment:
Missing 1 modification here : still says vendor/wimg

Reply:
That was a deliberate choice - this is about the `8.0.0` release and the changes regarding `installed_paths`. At the time 8.0.0 came out, PHPCompatibility didn't have the separate organization yet.

This info is relevant for users who are just now upgrading from an older PHPCompatibility version to 8.0.0 or later, so I figured that the users who need this information probably would still have the old repo location as well.

I'd be happy to change it, but am not sure if that wouldn't cause more confusion.
Alternatively, I could add `(vendor/phpcompatibility)` to it, so both options are shown ?